stubs/caldav/caldav/objects.pyi

```diff
@@ -221,6 +227,13 @@ class CalendarObjectResource(DAVObject):
     def add_organizer(self) -> None: ...
     def split_expanded(self) -> list[Self]: ...
     def expand_rrule(self, start: datetime.datetime, end: datetime.datetime) -> None: ...
+    def get_relatives(
+        self,
+        reltypes: Container[str] | None = None,
+        relfilter: Callable[[str], object] | None = None,
```

Comment:
Looks like `relfilter` is expected to return a `bool`: https://github.com/python-caldav/caldav/compare/v1.2.1...v1.3.5#diff-491a5dd4eece9b7ffe85041fb307fce0c648d69c707470fd697dbe2960a9ad7eR1869-R1870

```suggestion
        relfilter: Callable[[str], bool] | None = None,
```

The fact that `rel` is explicitly `str`'d here also indicates to me that `rel` (the object being passed to `relfilter`) might not necessarily be a `str`: https://github.com/python-caldav/caldav/compare/v1.2.1...v1.3.5#diff-491a5dd4eece9b7ffe85041fb307fce0c648d69c707470fd697dbe2960a9ad7eR1874

_But_ I'm not sure what the type _should_ be

Reply:
bool is correct

Reply:
Sorry that this is lacking both documentation, tests and example code.

Reply:
@tobixen, what's the expected type of the `rel` object that's _passed_ to the `relfilter` callable? Is it okay for the `relfilter` callable to assume that the object is going to be a `str`? Or should the parameter type be something more vague, like `object`?

Reply:
I regret creating this parameter at all ... I don't remember why I added it, it's not really needed (if one needs to do filtering, one can always do it after calling the function), there is no tests and no documentation.

Anyway, rel passed is a `icalendar.vText` attribute from the icalendar library.

Reply:
Thanks so much for the help @tobixen <3

We can't have `icalendar` as a dependency for these stubs here in typeshed, unfortunately (as it's not `py.typed`), so the type hint we should use here for now should probably be:

```suggestion
        relfilter: Callable[[Any], bool] | None = None,
```